deploy/handler/cluster_role.yaml

```
@@ -0,0 +1,19 @@
+{{if .ClusterReaderExists}}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: k8s-nmstate-project
```

The ` is` implicit and `project`  means nothing.
```suggestion
  name: nmstate-cluster-reader
```